target/linux/ramips/image/mt7621.mk

```diff
@@ -222,6 +222,15 @@ define Device/buffalo_wsr-600dhp
 endef
 TARGET_DEVICES += buffalo_wsr-600dhp
 
+define Device/cudy_wr1300
+  DTS := WR1300
+  IMAGE_SIZE := 15872k
+  SUPPORTED_DEVICES += cudy-wr1300 wr1300
```

Comment:
This is not needed for OpenWrt, but I'm always fascinated how people invent values for this property.
What's the string for "OEM OpenWrt" as discussed in the installation instructions ...

Reply:
Assuming I checked this correctly (see below), they have the board name set to cudy-wr1300. I renamed the device to cudy-wr1300 (changed hyphen to underscore) so that the board type matches exactly. This is consistent with the naming of some of the xiaomi routers, and will make upgrading from their image easier (no force needed), and remove the need for this unnecessary variable.

```
strings WR1300-OpenWRT19.07-0716.bin | grep supported_devices
{  "metadata_version": "1.0", "supported_devices":["cudy-wr1300"], "version": { "dist": "OpenWrt", "version": "19.07.3", "revision": "r11063-85e04e9f46", "target": "ramips/mt7621", "board": "cudy-wr1300" } }
```